uber/templates/registration/attendee_upload.html

```diff
@@ -17,17 +17,14 @@
         <div class="control-group"><h4>Imported Attendees</h4></div>
     <table class="table table-striped datatable">
 <thead><tr>
-    <th> <a href="index?order={{ order.last_first }}">Name</a> </th>
-    <th data-hide="phone"> <a href="index?order={{ order.badge_num }}">Badge</a> </th>
-    <th data-hide="phone"> <a href="index?order={{ order.paid }}">Paid</a> </th>
-    <th data-hide="phone">Age</th>
-    {% if c.AT_THE_CON %}
```

Comment:
This page was a pretty bad copy-paste job that I did a long time ago, so I removed a lot of the extra garbage from here while I was fixing up tables.